jenkins/job-configs/kubernetes-jenkins/node-e2e.yaml

```diff
@@ -89,7 +89,7 @@
             browser: githubweb
             browser-url: 'https://github.com/{repoName}'
             branches:
-                - 'master'
+                - '{branch}'
```

Comment:
You can set default to `master`.
Default value can be set like [this](https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/continuous-docker-validation.yaml#L70-L72)